### PR TITLE
Users can rebook appointments

### PR DESCRIPTION
**Pull request by AndrewVos**
# New button:

# Details are copied over:

### CODE REVIEW

File: app/controllers/appointments_controller.rb [hunk 1]
```diff
@@ -131,4 +131,12 @@ def redirect_after_successful_update
       redirect_to calendar_path, success: 'Appointment has been modified'
     end
   end
+
+  def copy_appointment
+    return nil unless params[:copy_from].present?
+    appointment = Appointment.find(params[:copy_from]).dup
+    appointment.start_at = nil
+    appointment.end_at = nil
+    appointment
```
Comment (benbarnett) [inline app/controllers/appointments_controller.rb]: Is there any other stuff like this? Associated guider maybe? Activity?

Comment (AndrewVos) [inline app/controllers/appointments_controller.rb]: The other stuff isn't rendered as inputs in the view, so they can't get saved. The only two afaik are those, cause they're actually hidden fields in the view.

Comment (benbarnett) [inline app/controllers/appointments_controller.rb]: Ah yeah, ok cool

**Comment by benbarnett** [comment]
😍

**Pull request merged**